Dev/NugetDownloadCount.5h.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Tony Clark
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# <xbar.title>NuGet Package Download Count</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Tony Clark</xbar.author>
+# <xbar.author.github>TheCriticalPath</xbar.author.github>
+# <xbar.desc>Displays the download count for each version of a specified nuget package.</xbar.desc>
+# <xbar.image></xbar.image>
```

Comment:
Could you upload your screenshot to a public host and put the URL here?

Reply:
image uploaded and file updated.